src/app/application.cpp

```diff
@@ -97,19 +98,33 @@ Application::Application(const QString &id, int &argc, char **argv)
     : BaseApplication(id, argc, argv)
     , m_running(false)
     , m_shutdownAct(ShutdownDialogAction::Exit)
+    , m_commandLineArgs(parseCommandLine(this->arguments()))
 {
+    validateCommandLineParameters();
+
+    setApplicationName("qBittorrent");
+
+    QString profileDir = m_commandLineArgs.portableMode ?
```

Comment:
I am thinking we should following the coding guidelines.
I am thinking the code uniformity adds readability.